ui/common/MiningStatistic.tsx

```diff
@@ -19,57 +19,51 @@ export default function InfoBox({
 }) {
   return (
     <div className="mt-5 flex items-center justify-between gap-x-2.5 text-center font-space-mono">
-      <div
-        className={clsx(
-          'h-[72px] w-full rounded-[5px] border-2 border-transparent bg-black/15  py-2.5 font-space-mono font-semibold',
-          {
-            'animate-duration-200 animate-pulse !border-[#FBEBC6] shadow-[0_0px_8px_rgba(251,235,198,0.5)]':
-              transactionHighlight === true,
-          }
-        )}
-      >
-        <div
-          className={clsx('text-2xl font-normal', {
-            'text-black/25': transaction === 0,
-            'text-white': transaction !== 0,
-          })}
-        >
-          {Intl.NumberFormat().format(transaction)}
-        </div>
-        <div
-          className={clsx('font-nunito text-[13px] font-bold', {
-            'text-black/25': transaction === 0,
-            'text-[#EDA081]': transaction !== 0,
-          })}
-        >
-          {transactionTitle}
+      <div className="relative h-[72px] w-1/2">
+        <div className="h-[72px] w-full rounded-[5px] border-2 border-transparent bg-black/15  py-2.5 font-space-mono font-semibold">
+          <div
+            className={clsx('text-2xl font-normal', {
+              'text-black/25': transaction === 0,
+              'text-white': transaction !== 0,
+            })}
+          >
+            {Intl.NumberFormat().format(transaction)}
+          </div>
+          <div
+            className={clsx('font-nunito text-[13px] font-bold', {
+              'text-black/25': transaction === 0,
+              'text-[#EDA081]': transaction !== 0,
+            })}
+          >
+            {transactionTitle}
+          </div>
         </div>
-      </div>
-      <div
-        className={clsx(
-          'h-[72px] w-full rounded-[5px] border-2 border-transparent bg-black/15 p-4 px-[15px] py-2.5 font-space-mono ',
-          {
-            'animate-duration-200 animate-pulse  !border-[#FBEBC6] shadow-[0_0px_8px_rgba(251,235,198,0.5)]':
-              bitcoinHighlight === true,
-          }
+        {transactionHighlight && (
+          <div className="absolute inset-0 h-[72px] w-full animate-pulse rounded-[5px] border-2 border-[#FBEBC6] shadow-[0_0px_8px_rgba(251,235,198,0.5)]"></div>
         )}
-      >
-        <div
-          className={clsx('text-2xl font-normal', {
-            'text-black/25': bitcoin === 0,
-            'text-white': bitcoin !== 0,
-          })}
-        >
-          {bitcoin.toFixed(4)}
-        </div>
-        <div
-          className={clsx('font-nunito text-[13px] font-bold', {
-            'text-black/25': bitcoin === 0,
-            'text-[#EDA081]': bitcoin !== 0,
-          })}
-        >
-          {bitcoinTitle}
+      </div>
+      <div className="relative h-[72px] w-1/2">
+        <div className="h-[72px] w-full rounded-[5px] border-2 border-transparent bg-black/15 p-4 px-[15px] py-2.5 font-space-mono ">
```

Comment:
`72` is divisible by `4` so let's use h-18, if tailwind doesnt support this we can add this to our `tailwind.config.js` in the `extend` object.

Reply:
it goes from h-16 to h-20 to h-24. I will add h-18 (72px) as might get used in the future